.github/workflows/release-github.yaml

```
@@ -24,9 +24,10 @@ jobs:
         run: |
           cd deploy/helm
           yq -i '.image.repository = "${{ env.GHCR_REPO }}"' grafana-operator/values.yaml
+          HELM_CHART_VERSION=$(cat grafana-operator/Chart.yaml | yq -r '.version')
```

```suggestion
          HELM_CHART_VERSION=$(yq -r '.version' grafana-operator/Chart.yaml)
```
That's a bit simpler :)